Makefile

```diff
@@ -254,9 +254,9 @@ diagrams: ## Build proposal diagrams
 	$(MAKE) -C docs/proposals/images
 
 .PHONY: book
-book: ## Build the GitBook
-	./hack/build-gitbook.sh
+book: ## Build the book
+	./hack/install-and-build.sh
```

Comment:
I don't think this path is correct

Reply:
You're right, also I think we can get rid of this target